src/Export.php

```diff
@@ -94,10 +94,14 @@ private function buildContent(?string $contentType)
             /** @var Content $record */
             foreach ($contentEntities as $record) {
                 $currentITem = $record->toArray();
+                // Get the select fields that have an entity referenced
+                $selectFields = $this->getSelectFields($record);
+                // Update the $currentItem with the right data that will be needed to make the reference when importing
+                $currentITem = $this->updateSelectFields($currentITem, $selectFields);
```

Comment:
Looks good! 

Should it be `currentItem` everywhere instead of `currentITem` @nestordedios ?

Reply:
I think it should be `currentItem` everywhere, I used `currentITem` as it was defined on line 96.

I will make the modifications and push a new commit.

Reply:
@I-Valchev , I have renamed the variable and pushed a new commit

Reply:
Nice, looks good! @bobdenotter can you review and merge?